BLSQ/iaso
# IA-3045: Registry: Pre-select a form in the lower part of the screen as well

Just like we have on the right hand side, we should also have a form (reference form if available, otherwise any other form) pre selected in the part below the map

Related JIRA tickets : IA-3045
## Self proofreading checklist


## Changes

- fetch org unit type detail to get reference forms of selected type
- pre-select a form 

## How to test

Make sure you have org unit type with a reference form and submission made with this form on an org unit of this type
Visit the parent of this org unit, it should pre-select form selection automatically.

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/e2705d3e-3725-4a45-a369-6dfce59e76d9
